_layouts/hs-apis-page.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+header_border: true
+---
+
+{% include breadcrumb.html
+  parent_permalink=page.portfolio-permalink
+  parent_title=page.portfolio
+%}
+
+<section class="page-tag-results">
+  <section class="background-dark usa-section section-intro">
+    <div class="grid-container">
+      <div class="section-intro-header">
+          <h1 class="section-heading section-heading-alt">10x</h1>
+          <h2>Eligibility APIs Initiative</h2>
+      </div>
+    </div>
+  </section>
+
+  <section class="grid-container usa-section content-wide">
```

Comment:
Would it make sense to give this some kind of element on top to make it clear that it's the side nav? I was thrown for a minute by seeing "background" and "background" next to each other before I figured it out.

Reply:
@stvnrlly Thanks for catching this, added code comments to make this clearer!